thrift/src/main/thrift/atoms/timeline.thrift

```diff
@@ -6,7 +6,7 @@ include "entity.thrift"
 include "../shared.thrift"
 
 struct TimelineItem {
-  1: required string title
+  1: optional string title
```

Comment:
Ah I see, clients on the old version will still be expecting a mandatory `title`